src/MultiSenseWordEmbeddingModel.scala

```diff
@@ -1,239 +0,0 @@
-import cc.factorie.model.{Parameters, Weights}
-import cc.factorie.optimize.{Trainer, AdaGradRDA}
-import cc.factorie.la.DenseTensor1
-import cc.factorie.util.Threading
-import java.io.{File, PrintWriter}
-import java.util.zip.GZIPInputStream
-import java.io.FileInputStream
-import java.io.OutputStreamWriter
-import java.util.zip.GZIPOutputStream
-import java.io.BufferedOutputStream
-import java.io.FileOutputStream
-
-abstract class MultiSenseWordEmbeddingModel(val opts: EmbeddingOpts) extends Parameters {
```

Comment:
Interface for all multi sense models (e.g. MSSG, NPMSSG). They share vocab building, model persistence. Seems to me that the shared `learnEmbeddings` method is a wrapper for a model's threaded `processExamples` implementation.